execution_engine/src/storage/error/mod.rs

```diff
@@ -1,4 +1,9 @@
+//! Storage errors.
+
+/// Errors for In-Memory storage implementation.
```

Comment:
As you and I discussed, perhaps this can be `#[cfg(test)]`?

Reply:
We talked more, and in-memory global state is going to be useful for perf profiling purposes.